packages/terra-avatar/src/Avatar.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import styles from './Avatar.scss';
+
+const cx = classNames.bind(styles);
+
+const AvatarVariants = {
+  FACILITY: 'facility',
+  USER: 'user',
+};
+
+const propTypes = {
+  /**
+   * The text content that specifies the alternative text for the image.
+   */
+  alt: PropTypes.string,
+  /**
+   * The image to display.
+   */
+  image: PropTypes.string,
+  /**
+   * The initials to display. Can be two or three characters, otherwise a fallback icon displays.
+   */
+  initials: PropTypes.string,
+  /**
+   * The avatar variant. One of `AvatarVariants.FACILITY`, `AvatarVariants.USER`.
+   */
+  variant: PropTypes.oneOf([AvatarVariants.FACILITY, AvatarVariants.USER]),
+};
+
+const defaultProps = {
+  variant: AvatarVariants.USER,
+};
+
+const propsWarningMsg = 'Only one of the props: [image, initials] should be supplied.';
+
+const Avatar = ({
+  alt,
+  image,
+  initials,
+  variant,
+  ...customProps
+  }) => {
+  // Checks to run when not in production
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
trying to understand the reason for this?

Reply:
So we don't try to log to the console in prod.

Reply:
I see that propsWarningMsg is used only once. For that case, I recommend not using a variable

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/901237e7a6938c3655ffa65a772ace2bb6d7144e).